server/app/views/style/AdminStyles.java

```diff
@@ -51,6 +51,15 @@ public final class AdminStyles {
           "text-gray-500",
           StyleUtils.hover("bg-gray-200", "text-gray-900"));
 
+  public static final String DELETE_BUTTON =
```

Comment:
nitpick: I think calling this just `DELETE_BUTTON` is a bit misleading, because most of our buttons have text in them (and most of our buttons can already be styled by something in `ButtonStyles.java`). My first thought when I read this variable name is that it's a style for a button that has both the delete icon and the text "Delete" in it (like what the button looked like before). Can we name this to be something like `DELETE_ICON` or `DELETE_ICON_BUTTON` so it's slightly clearer it's only an icon?